components/codec/src/byte.rs

```diff
@@ -391,6 +392,51 @@ impl MemComparableByteCodec {
             }
         }
     }
+
+    /// Returns whether `encoded` bytes is encoded from `raw`. Returns `false` if `encoded` is invalid.
+    pub fn is_encoded_from(encoded: &[u8], raw: &[u8]) -> bool {
```

Comment:
Prefer to separate it into different PRs. One to implement `is_encoded_from`, one to switch to `components/codec`.

Reply:
ok

Reply:
this code is copied from [5641](https://github.com/tikv/tikv/pull/5641)
why we don't implement `is_encoded_from` in `components/codec` this PR?
@breeswish

Reply:
@b41sh  implementing it for components/codec is supposed to be done after #5641 .

Reply:
ok